cli/enginecli/setup_drop.py

```diff
@@ -0,0 +1,59 @@
+import argparse
+
+from brownie import network
```

Comment:
setup.py of api enviroment not depend on brownie anymore.
Maybe move to cli dir?

Reply:
@Andrei-Dolgolev : This is in `cli/`.

Reply:
My miss i was think you cmmented out api because of it in api.